.github/workflows/newbuild-vcpkg.yml

```diff
@@ -0,0 +1,333 @@
+name: build-vcpkg
+
+env:
+  VCPKG_BINARY_SOURCES: "clear;nuget,GitHub,read"
+  VCPKG_NUGET_REPOSITORY: "https://github.com/hpcc-systems/vcpkg"
+
+on:
+  push:
+    tags:
+      - "newbuild_*"
+      - "community_*"
+      - "!community_8.10.*"
+      - "!community_8.8.*"
+      - "!community_8.6.*"
+      - "!community_8.4.*"
+      - "!community_8.2.*"
+      - "!community_8.0.*"
+      - "!community_7.*"
+
+jobs:
+  check-skip:
+    runs-on: ubuntu-20.04
+    outputs:
+      platform: ${{ steps.skip_check.outputs.platform }}
+    steps:
+      - id: skip_check
+        uses: hpcc-systems/github-actions/changed-modules@main
+        with:
+          github_token: ${{ github.token }}
+
+  build:
+    strategy:
+      matrix:
+        os:
+          [
+            "ubuntu-18.04",
+            "ubuntu-20.04",
+            "ubuntu-22.04",
+            "windows-2019",
+            "windows-2022",
+            "macos-11",
+            "macos-12",
+          ]
+        include:
+          - os: "ubuntu-18.04"
```

Comment:
minor: I don't think you need the os: list, if you have explictly listed each combination via an include.